DropDown/src/DropDown.swift

```diff
@@ -7,6 +7,26 @@
 //
 
 import UIKit
+fileprivate func < <T : Comparable>(lhs: T?, rhs: T?) -> Bool {
```

Comment:
These two functions should be removed. After investigation, there were added only for one use of operator `>` with an optional at line [570](https://github.com/AssistoLab/DropDown/blob/master/DropDown/src/DropDown.swift#L570):
```swift
barButtonItemCondition: if let _ = anchorView as? UIBarButtonItem {
	let isRightBarButtonItem = anchorView?.plainView.frame.minX > window.frame.midX

	guard isRightBarButtonItem else { break barButtonItemCondition }

	let width = self.width ?? fittingWidth()
	let anchorViewWidth = anchorView?.plainView.frame.width ?? 0
	let x = -(width - anchorViewWidth)

	bottomOffset = CGPoint(x: x, y: 0)
}
```

Which should actually be changed to:
```swift
barButtonItemCondition: if let anchorView = anchorView as? UIBarButtonItem {
	let isRightBarButtonItem = anchorView.plainView.frame.minX > window.frame.midX

	guard isRightBarButtonItem else { break barButtonItemCondition }

	let width = self.width ?? fittingWidth()
	let anchorViewWidth = anchorView.plainView.frame.width
	let x = -(width - anchorViewWidth)

	bottomOffset = CGPoint(x: x, y: 0)
}
```